less/src/adapt-defaults.less

```diff
@@ -1,3 +1,39 @@
+// IMPORTANT - THESE ARE THE DEFAULT VARIABLES / LESS FOR COMPATIBILITY WITH ADAPT OPEN SOURCE WHILST THEME IS IN WORKING PROCESS
+
+// THIS FILE WILL BE REMOVED BEFORE FINAL RELEASE - PLEASE COMMENT OUT VARIABLES AS THEY BECOME REDUNDANT
```

Comment:
i think we can't remove the file since some less variabels are only defined here and used in components less files. this will cause a less compile error.

eg @item-spacing is used in
  src\components\adapt-contrib-accordion\less\accordion.less
  src\components\adapt-contrib-hotgraphic\less\hotgraphic.less
  src\components\adapt-contrib-matching\less\matching.less